src/Views/Admin/DashboardWidgets/Reports.php

```diff
@@ -73,15 +68,11 @@ public function enqueue_scripts($base)
                 'allTimeStart' => $this->get_all_time_start(),
                 'currency' => give_get_currency(),
                 'testMode' => give_is_test_mode(),
+                'assets' => GIVE_PLUGIN_URL,
```

Comment:
@JoshuaHungDinh I notice that in the JS files, you are using it like that: `${assets}assets/dist/images/admin/white-external-icon.svg`

So, we can let it more clear replace this line with something like this: 

`'assetsUrl' => GIVE_PLUGIN_URL . 'assets/',`

And then we can use that in the front like that:

`${assetsUrl}dist/images/admin/white-external-icon.svg`

Reply:
Good thought! Thanks @glaubersilva Resolved: 7e8db152cdff4f6db93c7a83df6bca3e5113e6ea